content/automate/ManageIQ/Transformation/StateMachines/TransformationPlanRequestApproval.class/__methods__/deny_request.rb

```diff
@@ -2,4 +2,4 @@
 message = $evm.object['reason']
 $evm.log('info', "Request denied because of #{message}")
 request.message = message
-request.deny('admin', msg = message)
+request.deny('admin', message)
```

Comment:
@fdupont-redhat I do not see how this would do anything more than assign an unused variable `msg`.  The result of the assignment `msg = message` would return the content of the `message` variable and passed that as the second parameter to the `deny` method.

While this is a reasonable cleanup I am concerned that this is not actually fixing any bug.

Reply:
@gmcculloug You're completely right. Going too fast. The problem is more likely the name of the method in deny_request.yml... Fixed in my appliance in an overlay domain, but not in ManageIQ. My bad.